[Settings]
localip = 192.168.
blockfile = D:\PortableApps\Files\Programming\PYTHON-authlog\blocklist.dat
authfile = D:\PortableApps\Files\Programming\PYTHON-authlog\auth.log
failcount = 2
vncfile = D:\PortableApps\Files\Programming\PYTHON-authlog\vncserver-x11.log
restart_time = 00:10:10
autoblockusers = root,pi

